.circleci/config.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - save_cache:
           paths:
             - /home/circleci/conda
-          key: v1-dependencies        
+          key: v1-dependencies
```

Comment:
If our dependencies are changed, we should bump this up to reset the cache and ensure that it still builds on the CI.

Reply:
> we should bump this up to reset the cache

Honestly speaking really had not much knowledge of CircleCI but I know I need to change the version of Ruby for which I updated docker image to `cimg/ruby:2.7.1-node` due to lower version of bundler error while doing `bundle install`.

The CI is triggering and passing as well. Are we missing anything here and where should I need to look?

Reply:
The cache is with respect to the ruby dependencies, regardless of the container. The build is working because our previous Gemfile installs are cached under that key. The only way to rebuild the cache is to change the key, so we need to do this to ensure that your new Gemfile works. I’m sure it will be fine, but we need to check.